apps/datahub/src/app/record/record-metadata/record-metadata.component.html

```diff
@@ -117,19 +117,20 @@
     class="container-lg px-4 lg:mx-auto"
     *ngIf="displayDownload$ | async"
   >
-    <datahub-record-downloads class="block mt-5"></datahub-record-downloads>
+    <datahub-record-downloads
+      class="block mt-[4.5rem]"
+    ></datahub-record-downloads>
```

Comment:
I thought it was alway preferable to work with rem that px because of scaling. If the project is px I see no problem to wsitch to px.

Reply:
the TW classes use `rem` behind the scenes so it's not like it's never used; but I struggle to see a good arguments for using `rem` instead of `px` in this kind of case (i.e. things that will never scale)

Reply:
Ok, I'll use px from now.